full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts

```diff
@@ -258,8 +260,8 @@ describe("iModelHub iModelsHandler", () => {
     requestContext = new AuthorizedClientRequestContext(accessToken);
 
     (requestContext as any).activityId = "iModelHub iModelsHandler";
-    projectId = await utils.getProjectId(requestContext, "iModelJsTest");
```

Comment:
We were using a separate project due to the tests deleting random iModels in the other test project.  Does PR fix this issue?

Reply:
Yes. The test that this PR fixes is "should update iModel extents" and the changes here remove the call to iModelClient.iModel.delete method which deletes the random (actually not random but the oldest) iModel.